app/Models/Tag.php

```diff
@@ -332,6 +336,14 @@ public function showProfileInfo(): bool
         return false;
     }
 
+    /**
```

Comment:
Would be great to have a small description of the function of what it's used for like
```
/**
 * Determine if a tag is auto-applied to newly created entities
 * @return bool
 */
```